arpc/rpc_stats.C

```diff
@@ -75,19 +75,22 @@ namespace rpc_stats {
 
     for (size_t i = 1; (key = it.next (&value)); i++) {
       appendStat (line, *key, value);
-      output_line (i, prefix, line, false);
+      output_line (i, prefix, line, !is_oneline);
```

Comment:
Took me a minute to understand what was happening here – you help the `output_line` logic in case we ever did want to dump it in a condensed format.

I'd have been a fan of just removing that entirely, but then it would break backwards compatibility, so this looks like the best approach.